src/middle-end/index.js

```diff
@@ -0,0 +1,33 @@
+const Redux = require('redux');
+const { routerMiddleware: RouterMiddleware } = require('react-router-redux');
+const { middleware: { thunk, errorLogger }, ...M } = require('strange-middle-end');
```

Comment:
To fix your build issues, try using this instead for now:
```suggestion
const { middleware: { thunk, errorLogger }, ...M } = require('strange-middle-end/dist/strange-middle-end');
```

It looks like the ES module version has some issues with the peer dependencies.  Gotta look into what rollup is doing there.  The commonjs version seems to be working fine.  I also need to checkout why the sourcemaps aren't being used!

Reply:
P.S. this is good for @zemccartney to know, since he's working-out some of these kinks right now by integrating the middle-end into a project.

Reply:
Cool will try thanks!

Reply:
mmm! very cool, thanks for hashing this out and the heads-up 🙏

Reply:
This should be taken care of in v0.1.1 https://github.com/rollup/rollup-plugin-commonjs/issues/410

Reply:
All settled now with the new version and the require statements not digging into `/dist`